File: lang/regex/findall.pl
#!/usr/bin/env perl

use strict;
use v5.10;

my ($v, $c) = qw/[aeiou] [qwrtypsdfghjklzxcvbnm]/;
my @m = (<STDIN> =~ /(?<=$c)($v{2,})(?=$c)/ig);
say(@m ? join("\n", @m) : -1);

File: lang/regex/findall.py
#!/usr/bin/env python3

import re

v = '[aeiou]'
c = '[qwrtypsdfghjklzxcvbnm]'
r = re.compile('(?<=' + c + ')' + v + '{2,}(?=' + c + ')', re.I)
print('\n'.join(r.findall(input()) or ['-1']))
